inc/stream/namespace.php

```diff
@@ -88,3 +89,15 @@ function override_network_admin_bar_menu( WP_Admin_Bar $wp_admin_bar ) {
 		]
 	);
 }
+
+/**
+ * Add the Xray ID to the log item meta data.
+ *
+ * @param array $record The stream log record to filter.
+ *
+ * @return array
+ */
+function filter_wp_stream_record_array( $record ) : array {
+	$record['meta']['xray'] = XRay\get_root_trace_id();
```

Comment:
I think XRay can be switched off so you'll need to check the cloud module config here - double check but it'd be something like `Altis\get_config()['modules]['cloud']['xray']`, if that's false then just return the record early.

Reply:
You might also want to (or instead) do a `function_exists`, as that would be more reliable